### PR TITLE
Merge final release/20.6

### PR DESCRIPTION
**Pull request by AliSoftware**
- [x] `changelog/*.txt` files updated with latest translations
- [x] A lot of PlayStore metadata (especially `full_description.txt`) updated now that the polyglots came up with a translation for the new copy that fits the 4000 character limit 🎉 
- [x] `values*/strings.xml` updated with latest translations
- [x] Version bumped in `version.properties`
- [x] WPAndroid submitted to Google for review
- [x] JPAndroid submitted to Google for review

### CODE REVIEW

**Review by AliSoftware — COMMENTED** [review]
😱  While submitting the WordPress build, I (almost missed but) noticed that the `<en-US>` entry for the release notes on the Play Store Console submission page… was empty, so the app almost didn't get **English** Release Notes submitted to Google 😱 

Indeed, when you look at the diff in this PR, you can see that while for Jetpack in `fastlane/jetpack_metadata/android/en-US/changelogs/`, `1262.txt` got deleted and `1266.txt` got added instead, but for WordPress in `fastlane/metadata/android/en-US/changelogs/`, `1262.txt` got deleted without a new `1266.txt` file being created.

My theory is that this is because [`en-gb` didn't get a translation for release notes in GlotPress](https://translate.wordpress.org/projects/apps/android/release-notes/en-gb/default/) this time around, and since [we treat `en-US` using a dirty trick for WP by downloading `en-gb` and mapping it to `en-US`](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/fastlane/lanes/localization.rb#L229-L230), the fact that the key was not translated for `en-gb` lead to it not being part of the `current` set when downloaded from GlotPress, and the hack thus wasn't able to extract the source from that.

This is yet another reason why this ugly hack with the `source_locale` and `{ 'en-gb' => 'en-US' }` should disappear and be refactored (ref: pdnsEh-jJ-p2), to avoid not only the ugliness and hard to read logic/config at call site, but also to avoid such bugs in the future.

---

In the meantime, despite that `fastlane/metadata/android/en-US/changelogs/1266.txt` not being present in this PR, I've manually copy/pasted the English copy of the release notes inside the `<en-US>` tag in the Play Store Console before submitting for review.

File: fastlane/jetpack_metadata/android/iw-IL/full_description.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Jetpack ל-WordPress
 
 יצירה
 
-לרעיונות הנהדרים שלך מגיעה פינה חמה באינטרנט. אפליקציית Jetpack ל-Android מאפשרת לך לבנות אתרים וליצור בלוגים על התשתית של WordPress. כדאי להשתמש בה ליצירת האתר שלך.
+לרעיונות הנהדרים שלך מגיעה פינה חמה באינטרנט. אפליקציית Jetpack ל-Android מאפשרת לך ליצור אתרים ובלוגים על תשתית של WordPress. כדאי להשתמש בה ליצירת האתר שלך.
```
Comment (AliSoftware) [inline fastlane/jetpack_metadata/android/iw-IL/full_description.txt:7]: Confirmed to have been taken into account by PlayStore:
image

File: fastlane/jetpack_metadata/android/sv-SE/short_description.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Bygg en WordPress-webbplats, skapa inlägg, lägg till foton och se statistik!
```
Comment (AliSoftware) [inline fastlane/jetpack_metadata/android/sv-SE/short_description.txt:1]: Confirmed to have been taken into account by PlayStore:
image

File: fastlane/metadata/android/ar/full_description.txt [hunk 3]
```diff
@@ -1,45 +1,40 @@
-يضع تطبيق ووردبريس لنظام التشغيل Android قوة النشر على الويب في متناولك. إنَّه مُنشئ موقع على الويب وأكثر من ذلك بكثير!
```
Comment (AliSoftware) [inline fastlane/metadata/android/ar/full_description.txt:1]: Submission to PlayStore Console confirmed that this and all the other `fastlane/metadata/android/*/full_description.txt` for other locales have been taken into account:
PlayStore-20 6

**Comment by AliSoftware** [comment]
> While submitting the WordPress build, I (almost missed but) noticed that the `<en-US>` entry for the release notes on the Play Store Console submission page… was empty, so the app almost didn't get **English** Release Notes submitted to Google 😱

Actually the exact same issue happened for the `en-US` for new "Play Store Description" (which I also almost failed to notice). So I ended up:
 - Submitting the translations for `play_store_desc` and `release_note_206` for `en-gb` in GlotPress, by just using the same copy as the original `en-US`
 - Then re-running `bundle exec fastlane download_wordpress_metadata_strings version:20.6 build_number:1266`, which got those files for `en-US` back — incidentally confirming my suspicion that this was the cause of the bug

Which pushed a second `Update WordPress metadata translations for 20.6` commit [c43e5ed](https://github.com/wordpress-mobile/WordPress-Android/pull/17118/commits/c43e5ede9530dee1f2fc6a996a2f6ac738634b86) after all to fix the issue.

Still had to manually copy/paste the description to Play Store Console too (I guess I could have re-run `supply` but was faster to do manual pasting)

File: fastlane/metadata/android/ar/full_description.txt (hunk 3 above)
Comment (AliSoftware) [reply fastlane/metadata/android/ar/full_description.txt:1]: Ah! That made me realize that "English — en-US" was not part of that list 😱  — same bug as for the release notes above I guess ➡️  [Follow up below](https://github.com/wordpress-mobile/WordPress-Android/pull/17118#issuecomment-1235735651)

**Review by oguzkocer — APPROVED** [review]
@AliSoftware I am sorry you had to deal with all these issues.

**Pull request merged**